src/pl/plisql/src/pl_gram.y

```diff
@@ -478,19 +492,23 @@ decl_sect_block		: decl_stmts
 						plisql_IdentifierLookup = IDENTIFIER_LOOKUP_NORMAL;
 						$$.n_initvars = plisql_add_initdatums(&($$.initvarnos));
 					}
-decl_sect		: opt_block_label decl_or_not decl_stmts
+				;
+
+decl_sect		:  decl_or_not decl_stmts
 					{
 						plisql_IdentifierLookup = IDENTIFIER_LOOKUP_NORMAL;
-						$$.label	  = $1;
+						$$.label	  = NULL;
 						$$.n_initvars = plisql_add_initdatums(&($$.initvarnos));
 					}
-				| opt_block_label
+				|
 					{
-						$$.label	  = $1;
+						plisql_IdentifierLookup = IDENTIFIER_LOOKUP_NORMAL;
+						$$.label	  = NULL;
```

Comment:
No. Only the label that comes right after the function signature (or the first PL block) is discarded. All other pl blocks inside can have the labels. The function name works as the first blocks label as well.

Reply:
> No. Only the label that comes right after the function signature (or the first PL block) is discarded. All other pl blocks inside can have the labels. The function name works as the first blocks label as well.

I think it's ok because oracle desn't have lable support either.